lib/rules/jsx-no-bind.js

```diff
@@ -81,6 +81,12 @@ module.exports = {
         node.callee.property.name === 'bind'
       ) {
         return 'bindCall';
+      } else if (
+        nodeType === 'ConditionalExpression'
```

Comment:
I think that this if condition can fit on one line :-)

Reply:
I try to match the code style of the function, I can change it if needed